azurelinuxagent/common/protocol/util.py

```diff
@@ -57,7 +56,7 @@ def __getattr__(self, name):
         raise AttributeError("%s not a valid value" % name)
 
 
-prots = _nameset(("WireProtocol", "MetadataProtocol"))
+prots = _nameset(("WireProtocol", ))
```

Comment:
nit; Please remove the trailing ",".

Reply:
The trailing , ensures that a tuple is created... I tried to preserve the data shape to ensure that we didn't get weird values downstream from the type change.  Changing the type will cause it to enumerate as a string correct and break nameset up into letters...  Or maybe I mis tested that.